.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,4 +23,23 @@ Please respect our Code of Conduct, in short:
 - Being respectful of differing viewpoints and experiences.
 - Gracefully accepting constructive criticism.
 - Focusing on what is best for the community.
-- Showing empathy towards other community members.
\ No newline at end of file
+- Showing empathy towards other community members.
+
+--->
+
+## Description
+
+<!---
```

Comment:
+1 for commenting out the instructions

Just one suggestion: since this repo also does a mergeable check on versioning (i.e. HISTORY.md / "New version is not required", we might also want to include that info in the template.